docs/source/gen/flytectl_delete_execution-queue-attribute.rst

```diff
@@ -0,0 +1,111 @@
+.. _flytectl_delete_execution-queue-attribute:
+
+flytectl delete execution-queue-attribute
+-----------------------------------------
+
+Deletes matchable resources of execution queue attributes
+
+Synopsis
+~~~~~~~~
+
+
+
+Deletes execution queue attributes for given project and domain combination or additionally with workflow name.
+
+Deletes execution queue attribute for project and domain
+Here the command delete execution queue attributes for project flytectldemo and development domain.
+::
+
+ flytectl delete execution-queue-attribute -p flytectldemo -d development 
+
+
+Deleting execution queue attribute using config file which was used for creating it.
+Here the command deletes execution queue attributes from the config file cra.yaml
+eg:  content of cra.yaml which will use the project domain and workflow name for deleting the resource
```

Comment:
Can we note that the tags are optional in the file (since they're unread)?

Reply:
Done.
`Tags are optional in the file as they are unread during the delete command but can be kept as the same file can be used for get, update or delete `
Added to other commands aswell